packages/x-date-pickers-pro/src/internal/hooks/useDesktopRangePicker/useDesktopRangePicker.tsx

```diff
@@ -42,10 +41,8 @@ export const useDesktopRangePicker = <
   useLicenseVerifier('x-date-pickers-pro', releaseInfo);
 
   const {
-    slots: innerSlots,
```

Comment:
This hook was the only one to handle the `slots` / `components` migration.
But `DesktopDateRangePicker` already handles it so it was never used.